jwst/assign_wcs/assign_wcs_step.py

```diff
@@ -41,14 +41,17 @@ class AssignWcsStep(Step):
     """
 
     spec = """
+        slit_y_low = float(default=-.55)
+        slit_y_high = float(default=.55)
+        # The y-range of a slit, (0, 0) is the center.
```

Comment:
Would the spec be clearer or more concise if
```
slit_y_range = float_list(default=[-.55, .55]) # The y-range of a slit, (0, 0) is the center
```
Does the `float_list` ConfigObj type work with our command line usage?  Not sure.

Regardless, the `# comment` should got on the same line as the each spec entry so they get picked up in the help.

Reply:
`float_list` does not work. It should probably be fixed in a separate PR. Comment moved.